data/json/mutations.json

```diff
@@ -662,7 +662,7 @@
         "type" : "mutation",
         "id" : "LACTOSE",
         "name" : "Lactose Intolerance",
-        "points" : -1,
+        "points" : -2,
```

Comment:
It's the worst of all the intolerances as calcium has limited sources. How to balance it against the other intolerances then?